docs/hoon/twig/wut-test/hep-case.md

```diff
@@ -11,13 +11,23 @@ union, with no default.
 ## Expands to
 
 ```
+:loop
 :ifno  q
   :lost 
 :if  :fits(p.i.q p)
   q.i.q
 :moar(q t.q)
 ```
 
+```
+|-
+?.  q
+  :lost
+?:  ?=(p.i.q p)
+  q.i.q
+$(q t.q)
+```
+
 ## Syntax
```

Comment:
Github's not showing it here, but I replaced `:lost` with `~|(%mint-lost ...)` ...